app/models/header_lookup.rb

```diff
@@ -27,9 +27,9 @@ def pid_lookup_by_scheme(term="", scheme="")
     if term.blank? || scheme.blank?
       nil
     elsif scheme == :mesh
-      @@memoized_mesh[term] || mesh_term_pid_local_lookup(term) || mesh_term_pid_lookup(term)
```

Comment:
it seemed that the easiest thing to do here was to simply return nil and then react to that in the function that calls this one. I left the original functions `mesh_term_pid_lookup` and `lcsh_term_pid_lookup` in case we need to use those again, but we can remove them if need be since we can always look at old versions on git.